docs/install-bitbucket-server.md

```diff
@@ -2,6 +2,9 @@
 
 # Installing Frogbot on Bitbucket Server repositories
 
+| Important: Using this method isn't recommended for open source projects. Learn more about it in the README under [Security note for pull requests scanning](../README.md#-security-note-for-pull-requests-scanning). |
```

Comment:
Important: Using this method isn't recommended for open source projects. Learn more about it in the README under [Security note for pull requests scanning](../README.md#-security-note-for-pull-requests-scanning
-->
Important: Using Frogbot on Bitbucket Server using JFrog Pipelines or Jenkins isn't recommended for open source projects. Read more about it [here](../README.md#security-note-for-pull-requests-scanning).